sympy/calculus/euler.py

```diff
@@ -3,23 +3,24 @@
 Euler-Lagrange Equations for given Lagrangian.
 """
 from itertools import combinations_with_replacement
+
 from sympy import Function, sympify, diff, Eq, S, Symbol, Derivative
-from sympy.core.compatibility import (iterable, range)
+from sympy.core.compatibility import iterable, range
```

Comment:
This line causes pylint issue `Redefining built-in 'range' (redefined-builtin)`. I can leave it as it is or put `#noqa` here.

Was using [six](https://pypi.python.org/pypi/six) ever considered instead of using custom written module?